src/web/wms/src/main/java/org/geoserver/wms/web/data/OpenLayersPreviewPanel.html

```diff
@@ -19,6 +19,7 @@
         </li>
       </ul>
       <div wicket:id="olPreview" style="margin-bottom: 1em"/>
+      <div wicket:id="dialog"/>
```

Comment:
Part of same change?

Reply:
Unsure, but probably, the preview page in the style editor was just not working, and this is structural, it cannot have been failing just for me. The code is referencing a dialog element that's just not there in the panel, it's available in the page but the code cannot find it. Might be a side effect of changes in the base page, but could not confirm it. Leaving it to be on the safe side.
@tbarsballe git tells me you added this dialog reference in the code in August along with the style groups, do you know how it was working since the HTML did not have one?